Chinese translation

yierwuhui opened the pull request:
Translation completed. 

For headlines.txt and keywords.txt, I've added or deleted contents as appropriate.

I will send some queries via Upwork to you.

yierwuhui committed:
Add files via upload

yierwuhui committed:
Create for delete.txt

yierwuhui committed:
Add files via upload

yierwuhui committed:
Delete for delete.txt

surferjeff commented on messages/ZH.json:
```diff
@@ -0,0 +1,317 @@
+    "saved": "已保存 {name}",
+    "addStep": "添加步骤",
+    "saveSteps": "保存步骤"
+  },
```
3 days ago, I added a `saveButton`.  Could you also add that translation here?
https://github.com/surferjeff/pdfsnake-locales/commit/a115b67ff18395c6f9d3430c17e6d9b0d74f38ec

surferjeff reviewed (changes requested):
Looking very good.

yierwuhui commented:
Hi Jeffrey,

I guess the content of the string "saveButton" is "Save". And "Save" should be translated into "保存".

发件人: Jeffrey Rennie ***@***.***>
发送时间: 2021年7月6日 0:54
收件人: surferjeff/pdfsnake-locales ***@***.***>
抄送: yierwuhui ***@***.***>; Author ***@***.***>
主题: Re: [surferjeff/pdfsnake-locales] Chinese translation (#12)


@surferjeff requested changes on this pull request.

Looking very good.

________________________________

In messages/ZH.json<https://github.com/surferjeff/pdfsnake-locales/pull/12#discussion_r664051399>:

> +    "viewFirst": "查看前 {count} 页",



+    "viewPrevious": "查看此前的 {count} 页",



+    "pageRange": "{first}-{last}",



+    "viewNext": "查看此后的 {count} 页",



+    "viewFinal": "查看最后 {count} 页",



+    "recalculating": "正在重新计算……"



+  },



+  "home": {



+    "step": "第 {number} 步： {name}",



+    "help": "帮助",



+    "saveAs": "将这些步骤另存为：",



+    "cancel": "取消",



+    "saved": "已保存 {name}",



+    "addStep": "添加步骤",



+    "saveSteps": "保存步骤"



+  },



3 days ago, I added a saveButton. Could you also add that translation here?
a115b67<https://github.com/surferjeff/pdfsnake-locales/commit/a115b67ff18395c6f9d3430c17e6d9b0d74f38ec>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/surferjeff/pdfsnake-locales/pull/12#pullrequestreview-699331831>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUTK3EIGUHJEZXU6W3GWPO3TWHPTHANCNFSM472MCFNQ>.

yierwuhui committed:
Delete ZH.json

yierwuhui committed:
Add files via upload

yierwuhui commented:
Hi Jeffrey,

I found some redundant spaces and updated the file "zh.json" in my repository.

For the translation of the string "saveButton", please tell me where and how should I add this translation into the file if this addition is to be completed by me.


发件人: Jeffrey Rennie ***@***.***>
发送时间: 2021年7月6日 0:54
收件人: surferjeff/pdfsnake-locales ***@***.***>
抄送: yierwuhui ***@***.***>; Author ***@***.***>
主题: Re: [surferjeff/pdfsnake-locales] Chinese translation (#12)


@surferjeff requested changes on this pull request.

Looking very good.

________________________________

In messages/ZH.json<https://github.com/surferjeff/pdfsnake-locales/pull/12#discussion_r664051399>:

> +    "viewFirst": "查看前 {count} 页",



+    "viewPrevious": "查看此前的 {count} 页",



+    "pageRange": "{first}-{last}",



+    "viewNext": "查看此后的 {count} 页",



+    "viewFinal": "查看最后 {count} 页",



+    "recalculating": "正在重新计算……"



+  },



+  "home": {



+    "step": "第 {number} 步： {name}",



+    "help": "帮助",



+    "saveAs": "将这些步骤另存为：",



+    "cancel": "取消",



+    "saved": "已保存 {name}",



+    "addStep": "添加步骤",



+    "saveSteps": "保存步骤"



+  },



3 days ago, I added a saveButton. Could you also add that translation here?
a115b67<https://github.com/surferjeff/pdfsnake-locales/commit/a115b67ff18395c6f9d3430c17e6d9b0d74f38ec>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/surferjeff/pdfsnake-locales/pull/12#pullrequestreview-699331831>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUTK3EIGUHJEZXU6W3GWPO3TWHPTHANCNFSM472MCFNQ>.

yierwuhui committed:
Update ZH.json

yierwuhui commented:
Hi Jeffrey,

I've added "saveButton": "保存" into the zh.json file in my repository.
